packages/server/graphql/mutations/addReactjiToReactable.ts

```diff
@@ -69,11 +76,15 @@ const addReactjiToReactable = {
     const subOptions = {mutatorId, operationId}
 
     //AUTH
-    const dbTable = tableLookup[reactableType]
-    const reactable = (await r
-      .table(dbTable)
-      .get(reactableId)
-      .run()) as Reactable
+    let reactable: Reactable
+    const isPgTable = reactableType === 'RESPONSE'
```

Comment:
+1 super nitty, but if you do the lookup here (`const pgLoaderName = pgDataloaderLookup[reactableType]`) then you'll save a line & it'll future proof this bit of code for when we introduce more types.